homeassistant/components/velux/cover.py

```diff
@@ -60,8 +60,16 @@ def current_cover_position(self):
 
     @property
     def device_class(self):
-        """Define this cover as a window."""
-        return 'window'
+        """Define this cover as either window, blind, awning or rollershutter.""" 
+        from pyvlx.opening_device import Blind, RollerShutter, Window, Awning
+        if isinstance(self.node, Window):
+            return 'window'
+        if isinstance(self.node, Blind):
+            return 'blind'
+        if isinstance(self.node, RollerShutter):
+            return 'shutter'
+        if isinstance(self.node, Awning):
+            return 'awning'
```

Comment:
I think this should have an explicit `return None` at the end in case none of the if cases match